app/_src/gateway/production/monitoring/health-check.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Health Check
+content_type: tutorial
+---
+
+This tutorial will guide you through the process of using the new Node Readiness Endpoint, which provides a more reliable and simple way to determine if Kong is ready to serve user requests.
+
+To put it simply, health check endpoint will return a `200 OK` response when Kong is ready, or a `503 Service Unavailable` response when it's not. This is useful for load balancers and other tools that need to monitor the readiness of Kong instances.
```

Comment:
1. `To put it simply` feels like filler here. I think the sentence is perfectly clear without it
2. Subjectively, I think `returns` is more apt than `will return`, but `will return` is fine with me if you think it is better

```suggestion
The health check endpoint returns a `200 OK` response when Kong is ready, or a `503 Service Unavailable` response when it's not. This is useful for load balancers and other tools that need to monitor the readiness of Kong instances.
```